src/adjustfloatingpositionerfunctor.cpp

```diff
@@ -115,6 +115,7 @@ FunctorCode AdjustFloatingPositionersFunctor::VisitStaffAlignment(StaffAlignment
 
         // Handle within placement (ignore collisions for certain classes)
         if (place == STAFFREL_within) {
+            if (m_classId == CPMARK) continue;
```

Comment:
use `else if`?

Reply:
I agree it looks unusual. But since we continue on success it would make no difference and it is a bit more compact like that.